nl/guides/course-management/README.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Cursusbeheer
+description: "Tutorial: cursusbeheer"
+---
+
+# Cursusbeheer
+
+**Inhoudsopgave**
+[[toc]]
+
+Een cursus wordt opgebouwd als een [leerpad](#leerpad) met oefeningen die gebundeld worden in oefeningenreeksen. De opeenvolging van oefeningenreeksen impliceert een mogelijke volgorde waarin de oefeningen kunnen opgelost worden.
```

Comment:
```suggestion
Een cursus wordt opgebouwd als een leerpad met oefeningen die gebundeld worden in oefeningenreeksen. De opeenvolging van oefeningenreeksen impliceert een mogelijke volgorde waarin de oefeningen kunnen opgelost worden.
```
add the fact that a course contains users of which at least one is a course admin